book/02-git-basics/sections/recording-changes.asc

```diff
@@ -28,7 +28,7 @@ Your branch is up-to-date with 'origin/master'.
 nothing to commit, working directory clean
 ----
 
-This means you have a clean working directory – in other words, there are no tracked and modified files.
+This means you have a clean working directory – in other words, there are no modified files among tracked ones.
```

Comment:
Well, guys, I think my duty was to draw your attention to this ambiguity and your duty is to eliminate it